### DISCUSSION TITLE
Show Names on Image Message

### DISCUSSION THREAD

**Discussion by shamxeed** [flyerhq/react-native-chat-ui#48] [Ideas / Feature Requests]
I love everything about this Lib. It's very amazing!! But I was wondering why `ImageMessage` is not supporting showing names?
In a group chat, I enabled `showNames` prop so that messages can be identified by their authors but unfortunately `ImageMessage` is not supporting that and I think it's a bit weird because other group members will not be able to know who send the Photo.

SCREENSHORT:

![2022_03_18_14_24_IMG_3469](https://user-images.githubusercontent.com/57716026/159011517-43bb26e2-0ef1-4e1e-b1f1-71d8e507bca1.PNG)

Based on the context of the conversation above, we'll believe that the photo was shared by Nuzahd Khan but actually it wasn't sent by her!

**Comment by demchenkoalex** [comment]
Now when you put it this way seems like a valid issue. We were basing on a Telegram design where images do not show the name (it appears only on text messages) and how Telegram is fixing this - they also show avatars, if you enable those, there will be no issue in believing that Nazahd sent this photo. Can't suggest anything else right now, cause we don't really plan to change the initial design assumptions, cause there can be many. We just chose this way for now.
